server/src/test/java/org/elasticsearch/search/aggregations/AggregatorFactoriesBuilderTests.java

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
I think we have `BaseAggregationTestCase` for aggregators so we could maybe add the testing for this case directly there. It would test all implementations rather than just a few.